crates/gpui/src/platform/windows/text_system.rs

```diff
@@ -222,10 +220,11 @@ impl WindowsTextSystemState {
             .get_font_matches(Attrs::new().family(cosmic_text::Family::Name(name)));
         for font in family.as_ref() {
             let font = self.font_system.get_font(*font).unwrap();
-            if font.as_swash().charmap().map('m') == 0 {
-                self.font_system.db_mut().remove_face(font.id());
-                continue;
-            };
+            // TODO: figure out why this is causing fluent icons from loading
+            // if font.as_swash().charmap().map('m') == 0 {
+            //     self.font_system.db_mut().remove_face(font.id());
+            //     continue;
+            // };
```

Comment:
this is the same on linux, but it makes the fluent icons not load. could someone comment on the intention of this block? Seems kind of arbitrary checking for the 'm' in the charmap.

Reply:
I think this is probably fine, particularly with the comment.